dashboard/src/api/resources/Metrics.js

```diff
@@ -0,0 +1,30 @@
+const METRICS_API_URL = "http://localhost:9090/api/v1";
+
+export default class MetricsAPI {
+  fetchInstances() {
+    var fetchAddress = `${METRICS_API_URL}/label/__name__/values`;
+
+    return fetch(fetchAddress, {
+      headers: {
+        "Content-Type": "application/json",
+      },
+    })
+      .then((res) => res.json().then((json_data) => json_data))
+      .catch((error) => {
+        console.error(error);
+      });
+  }
+
+  fetchMetrics(instance) {
+    var fetchAddress = `${METRICS_API_URL}/metadata`;
+    return fetch(fetchAddress, {
+      headers: {
+        "Content-Type": "application/json",
+      },
+    })
+      .then((res) => res.json().then((json_data) => json_data))
```

Comment:
".then((json_data) => json_data)" this seems like an identity lambda that doesn't do anything, am I missing something?

Reply:
yea it's an artifact left over from debugging